USAGE_android.md

```diff
@@ -806,6 +807,10 @@ optional arguments:
                         setup for replay. The default without this option is to use a Virtual Swapchain
                         of images which match the swapchain in effect at capture time and which are
                         copied to the underlying swapchain of the implementation being replayed on.
+  --no-retry-on-timeout By default, Vulkan api calls that have a timeout parameter
```

Comment:
What is the use case where you want to prioritize reproducing the exact timeout values rather than correct synchronizations of the original content?